exercises/atbash-cipher/src/Atbash.hs

```diff
@@ -1,7 +1,7 @@
 module Atbash (decode, encode) where
 
 decode :: String -> String
-decode phrase = error "You need to implement this function."
+decode xs = error "You need to implement this function."
```

Comment:
@petertseng I agree with you for this case:
The meaning of the parameter is fixed and cannot vary (as it can for some type `[a]`, say). So for me it makes sense to use a more meaningful name, too.